census/src/main/java/io/grpc/census/CensusStatsModule.java

```diff
@@ -334,71 +413,83 @@ public void outboundMessage(int seqNo) {
      * (potentially racy) direct updates of the volatile variables.
      */
     static {
-      AtomicReferenceFieldUpdater<CallAttemptsTracerFactory, ClientTracer> tmpStreamTracerUpdater;
       AtomicIntegerFieldUpdater<CallAttemptsTracerFactory> tmpCallEndedUpdater;
       try {
-        tmpStreamTracerUpdater =
-            AtomicReferenceFieldUpdater.newUpdater(
-                CallAttemptsTracerFactory.class, ClientTracer.class, "streamTracer");
         tmpCallEndedUpdater =
             AtomicIntegerFieldUpdater.newUpdater(CallAttemptsTracerFactory.class, "callEnded");
       } catch (Throwable t) {
         logger.log(Level.SEVERE, "Creating atomic field updaters failed", t);
-        tmpStreamTracerUpdater = null;
         tmpCallEndedUpdater = null;
       }
-      streamTracerUpdater = tmpStreamTracerUpdater;
       callEndedUpdater = tmpCallEndedUpdater;
     }
 
-    private final CensusStatsModule module;
```

Comment:
I don't see why some of these fields are no longer private. I don't see any references outside of this class. This applies to some fields in ClientTracer as well. I see some stuff related to `attemptsState`, but that seems relatively few.

It actually seems like some of the older fields (e.g., `outboundMessageCount`) no longer need to be package-private, because the code moved to recordFinishedRpc.

Reply:
Yeah, some Factory fields can be private as they are copied to ClientTracer. Changed back. As for the fields in ClientTracer, because the class itself is already private so every field is essentially private, I don't bother to check every field if they are used outside, just assume all of them are and keep package-private to avoid synthetic accessors.